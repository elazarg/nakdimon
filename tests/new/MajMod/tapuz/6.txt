כְּדֵי שפרטים ידלפו מהדרכון הַבִּיּוֹמֶטְרִי שֶׁלְּךָ צְרִיכִים לִקְרוֹת הַרְבֵּה דְּבָרִים. הָרִאשׁוֹן: מִי שֶׁרוֹצֶה לִגְנוֹב אֶת הַפְּרָטִים שֶׁלְּךָ צָרִיךְ שֶׁתִּהְיֶה לוֹ גִּישָׁה לדרכון הַסְּפֵּצִיפִי שֶׁלְּךָ. הוּא לֹא יָכוֹל לִגְנוֹב אוֹתָם מֵהָאֲוִויר. אִם הוּא רוֹצֶה לִגְנוֹב פְּרָטִים שֶׁל 50,000 אִישׁ הוּא צָרִיךְ לְהַשִּׂיג גִּישָׁה ל 50,000 דרכונים/תעודות זֶהוּת. בַּמַּאֲגָר הוּא צָרִיךְ גִּישָׁה אַחַת - לַמַּאֲגַר. וְהוּא יָכוֹל לִגְנוֹב אֶת הַפְּרָטִים הביומטרים שֶׁל מֵאוֹת אַלְפֵי אֶזְרָחִים. בַּמִּישׁוֹר הַפְּרָטִי הַפְּגִיעָה הִיא לֹא רַק בפרטיות שֶׁלְּךָ, לְמָשָׁל הַיּוֹם בְּיוֹתֵר וְיוֹתֵר מְקוֹמוֹת אֲנָשִׁים מזדהים יוֹתֵר וְיוֹתֵר בְּצוּרָה בִּיוֹמֶטְרִית כְּמוֹ טְבִיעַת אֶצְבַּע. אִם מִישֶׁהוּ מַשִּׂיג אֶת טְבִיעַת הָאֶצְבַּע שֶׁלְּךָ הוּא יָכוֹל לַעֲשׂוֹת אִיתָּהּ הֲמוֹן דְּבָרִים, מלרוקן לְךָ אֶת קופת הַפֶּנְסִיָּה דֶּרֶךְ גִּישָׁה לַתִּיק הָרְפוּאִי שֶׁלְּךָ וְעַד הִתְחַזּוּת אֵלַיִיךְ כשמבצע עֲבֵירָה פְּלִילִית. בַּמִּישׁוֹר הַצִּיבּוּרִי דליפה שֶׁל מַאֲגַר הַמֵּידָע יְכוֹלָה לִהְיוֹת פְּגִיעָה חֲמוּרָה בְּבִטָחוֹן מְדִינַת יִשְׂרָאֵל.