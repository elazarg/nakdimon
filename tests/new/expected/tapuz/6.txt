כְּדֵי שֶׁפְּרָטִים יִדְלְפוּ מֵהַדַּרְכּוֹן הַבִּיּוֹמֶטְרִי שֶׁלָּךְ צְרִיכִים לִקְרוֹת הַרְבֵּה דְּבָרִים.
הָרִאשׁוֹן: מִי שֶׁרוֹצֶה לִגְנוֹב אֶת הַפְּרָטִים שֶׁלָּךְ צָרִיךְ שֶׁתִּהְיֶה לוֹ גִּישָׁה לַדַּרְכּוֹן הַסְּפֵּצִיפִי שֶׁלָּךְ.
הוּא לֹא יָכוֹל לִגְנוֹב אוֹתָם מֵהָאֲוִויר.
אִם הוּא רוֹצֶה לִגְנוֹב פְּרָטִים שֶׁל 50,000 אִישׁ הוּא צָרִיךְ לְהַשִּׂיג גִּישָׁה לְ 50,000 דַּרְכּוֹנִים/תְּעוּדוֹת זֶהוּת.
בַּמַּאֲגָר הוּא צָרִיךְ גִּישָׁה אַחַת - לַמַּאֲגָר.
וְהוּא יָכוֹל לִגְנוֹב אֶת הַפְּרָטִים הַבִּיּוֹמֶטְרִים שֶׁל מֵאוֹת אַלְפֵי אֶזְרָחִים.
בַּמִּישׁוֹר הַפְּרָטִי הַפְּגִיעָה הִיא לֹא רַק בַּפְּרָטִיּוּת שֶׁלָּךְ, לְמָשָׁל הַיּוֹם בְּיוֹתֵר וְיוֹתֵר מְקוֹמוֹת אֲנָשִׁים מִזְדַּהִים יוֹתֵר וְיוֹתֵר בְּצוּרָה בִּיּוֹמֶטְרִית כְּמוֹ טְבִיעַת אֶצְבַּע.
אִם מִישֶׁהוּ מַשִּׂיג אֶת טְבִיעַת הָאֶצְבַּע שֶׁלָּךְ הוּא יָכוֹל לַעֲשׂוֹת אִיתָּהּ הֲמוֹן דְּבָרִים, מִלְּרוֹקֵן לְךָ אֶת קוּפַּת הַפֶּנְסְיָה דֶּרֶךְ גִּישָׁה לַתִּיק הָרְפוּאִי שֶׁלָּךְ וְעַד הִתְחַזּוּת אֵלַיִיךְ כְּשֶׁמְּבַצֵּעַ עֲבֵירָה פְּלִילִית.
בַּמִּישׁוֹר הַצִּיבּוּרִי דְּלִיפָה שֶׁל מַאֲגַר הַמֵּידָע יְכוֹלָה לִהְיוֹת פְּגִיעָה חֲמוּרָה בְּבִטְחוֹן מְדִינַת יִשְׂרָאֵל.
